README.md

```diff
@@ -139,6 +139,46 @@ Any generated local resource will have stored internally the same client used to
 made against the resource will automatically use the same client. This will prevent potential issues of accidentally
 using the wrong API key when interacting with a resource in a multi-client environment.
 
+## Parameters (BETA):
+
+Most functions in this library accept a `Dictionary<string, object>` as their sole parameter, which is ultimately used as the body of the HTTP request against EasyPost's API. If you instead would like to use .NET objects to construct API call parameters, you can use the various `Parameters` classes (currently in beta).
+
+For example, to create an address:
+
+```csharp
+// Use an object constructor to create the address creation parameters
+var addressCreateParameters = new EasyPost.BetaFeatures.Parameters.Addresses.Create {
+    Name = "My Name",
+    Street1 = "123 Main St",
+    City = "San Francisco",
+    State = "CA",
+    Zip = "94105",
+    Country = "US",
+    Phone = "415-123-4567"
+};
+
+// You can add additional parameters as needed outside of the constructor
+addressCreateParameters.Company = "My Company";
+
+// Then convert the object to a dictionary
+// This step will validate the data and throw an exception if there are any errors (i.e. missing required parameters)
```

Comment:
Correct me if I am wrong. Currently, all the parameters are optional, so I don't think an exception will be thrown in `ToDictionary` method.

Reply:
Yes, all parameters are optional right now, to keep this PR simple. In the next PRs, we'll actually start making some required.